test/data-basic.jl

```diff
@@ -189,53 +189,74 @@ end
         end
     end
 
-    @testset "test basic jacobian" begin  
+    @testset "test basic jacobian" begin
+        @testset "5-bus-case" begin
+            data = make_basic_network(PowerModels.parse_file("../test/data/matpower/case5.m"))
+            for (i, bus) in data["bus"] # All buses PQ
+                bus["bus_type"] = 1
+            end
+            J = PowerModels.calc_basic_jacobian_matrix(data);
+
+            num_bus = length(data["bus"])
+            
+            @test size(J)[2] == num_bus * 2
+            @test size(J)[1] == num_bus * 2
+            @test isapprox(LinearAlgebra.det(J), 1524.5588; atol=1e-4)
```

Comment:
The `LinearAlgebra.det` is not a good full matrix metric.
``` julia
J = calc_basic_jacobian_matrix(data) # J :: SparseMatrixCSC
J |> det # 1524.55...
J |> Matrix |> det # -1590.68...
U, V = svd(J |> Matrix); reduce(*, V) # 3967.00...
```

What do you think about something simple like:
``` julia
J = calc_basic_jacobian_matrix(data) # J :: SparseMatrixCSC
metric(arr) = reduce(*, [sum(row) for row in eachcol(arr)])
J |> metric # -132063.9683811387
J |> Matrix |> metric # -132063.9683811387
```

Reply:
I an fine with the sum metric instead.